### PR TITLE
[Bulk] Validation Fixes

### PR DESCRIPTION
**Pull request by AqMoh**
This PR fixes issues with invalid HTML attributes, missing descriptions, missing column headers, and other errors.

### CODE REVIEW

File: BingMaps/rest-services/directly-accessing-the-bing-maps-tiles.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Directly accessing the Bing Maps tiles | Microsoft Docs"
+title: "Directly Accessing the Bing Maps Tiles | Microsoft Docs"
+description: "This article outlines direct access the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Maps control."
```
Comment (MeeraDi) [inline BingMaps/rest-services/directly-accessing-the-bing-maps-tiles.md:3]: This article outlines direct access to the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Map control.

File: BingMaps/rest-services/examples/driving-route-example.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Driving Route Example | Microsoft Docs"
+description: "The article provides a driving route example and shows how to request a driving route between two locations that minimizes the use of toll roads. Responses are shown for both XML and JSON formats."
```
Comment (MeeraDi) [inline BingMaps/rest-services/examples/driving-route-example.md:3]: Responses are shown for both XML and JSON.

File: BingMaps/rest-services/examples/distance-matrix-histogram-example.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Distance Matrix Histogram Example | Microsoft Docs"
+description: "This distance matrix histogram example returns a distance matrix histogram between a set of origins and destinations between a specific period of time."
```
Comment (MeeraDi) [inline BingMaps/rest-services/examples/distance-matrix-histogram-example.md:3]: for a specific period of time.

File: BingMaps/rest-services/examples/driving-route-with-route-path-example.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Driving Route with Route Path Example | Microsoft Docs"
+description: "This example returns a driving route from the Minneapolis, Minnesota to St. Paul, Minnesota. Results are shown for both JSON and XML response formats."
```
Comment (MeeraDi) [inline BingMaps/rest-services/examples/driving-route-with-route-path-example.md:3]: This example returns a driving route from Minneapolis, Minnesota to St. Paul, Minnesota.

File: BingMaps/rest-services/examples/driving-route-with-route-path-example.md [hunk 5]
```diff
@@ -16,7 +17,7 @@ ms.service: "bing-maps"
 
 # Driving Route with Route Path Example
 
-This example returns a driving route from the Minneapolis, Minnesota to St. Paul, Minnesota. The route is optimized for distance and requests point values for the route’s path. Results are shown for both JSON and XML response formats..  
+This example returns a driving route from the Minneapolis, Minnesota to St. Paul, Minnesota. The route is optimized for distance and requests point values for the route’s path. Results are shown for both JSON and XML response formats.
```
Comment (MeeraDi) [inline BingMaps/rest-services/examples/driving-route-with-route-path-example.md:20]: his example returns a driving route from Minneapolis, Minnesota to St. Paul, Minnesota.

**Comment by AqMoh** [comment]
**Global effort to fix build validation errors**
@stevemunk 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

File: BingMaps/rest-services/getting-traffic-incident-data.md [hunk 6]
```diff
@@ -24,7 +25,7 @@ You can get traffic incident information along a route using the [Routes](routes
   
  For traffic coverage by country, see [Bing Maps Geographic Coverage](../coverage/geographic-coverage.md).  
   
-## Traffic incident information along a route (Routes API)
+## Traffic Incident Information Along a Route (Routes API)
```
Comment (stevemunk) [inline BingMaps/rest-services/getting-traffic-incident-data.md:28]:
```suggestion
## Traffic incident information along a route (Routes API)
```

File: BingMaps/rest-services/getting-traffic-incident-data.md [hunk 7]
```diff
@@ -147,7 +148,7 @@ The following is an example of an itinerary item that contains both warnings and
 }  
 ```  
   
-## Traffic incident information within a geographical area (Traffic API)
+## Traffic Incident Information within a Geographical Area (Traffic API)
```
Comment (stevemunk) [inline BingMaps/rest-services/getting-traffic-incident-data.md:151]:
```suggestion
## Traffic incident information within a geographical area (Traffic API)
```

File: BingMaps/rest-services/common-parameters-and-types/type-identifiers/index.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
 ---
 # required metadata
 title: "Type Identifiers | Microsoft Docs"
-description: An overview page listing the top-level type identifiers used by the Bing Maps APIs.
-author: "v-chrfr"
-ms.author: v-munksteve
+description: "This Type identifiers article lists top-level type IDs used by the Bing Maps APIs."
```
Comment (stevemunk) [inline BingMaps/rest-services/common-parameters-and-types/type-identifiers/index.md:4]:
```suggestion
description: The overview page for the Type Identifiers section contains links to the top-level types used by the Bing Maps API including EatDrink, SeeDo, Shop, BanksAndCreditUnions, Hospitals, HotelsAndMotels and Parking.
```

File: BingMaps/rest-services/developer-resources.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "REST Developer Resources | Microsoft Docs"
+description: "This article provides REST developer resources to learn more about Bing Maps, get your questions answered, and get the latest announcements."
```
Comment (stevemunk) [inline BingMaps/rest-services/developer-resources.md:3]:
```suggestion
description: This article provides links to Bing Maps REST developer resources such as forums, blogs, social media accounts and links to information on account issues and licensing questions.
```

File: BingMaps/rest-services/directly-accessing-the-bing-maps-tiles.md [hunk 10]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Directly accessing the Bing Maps tiles | Microsoft Docs"
+title: "Directly Accessing the Bing Maps Tiles | Microsoft Docs"
+description: "This article outlines direct access to the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Maps control."
```
Comment (stevemunk) [inline BingMaps/rest-services/directly-accessing-the-bing-maps-tiles.md:3]:
```suggestion
description: Learn how to access the Bing Map tiles directly to add Bing Maps as a tile layer in a non-Bing Maps control."
```

File: BingMaps/rest-services/elevations/get-elevations.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Get Elevations | Microsoft Docs"
+description: "This article provides URLs that let you get elevation values (in meters) for a set of locations, a polyline path, or area on the Earth."
```
Comment (stevemunk) [inline BingMaps/rest-services/elevations/get-elevations.md:3]:
```suggestion
description: "This article provides URLs that let you get elevation values (in meters) for a set of locations, a polyline path, or area on the Earth. This includes getting elevations for latitude and longitude coordinates, at equally-spaced locations along a polyline path or within a bounding box, getting the offset of the geoid sea level earth model from the ellipsoid earth model, also an explanation about elevations and coordinate values and working with earth models and zoom level." 
```

File: BingMaps/rest-services/elevations/index.md [hunk 12]
```diff
@@ -1,5 +1,6 @@
 ---
-title: "Elevations API | Microsoft Docs"
+title: "Bing Maps Elevations API | Microsoft Docs"
+description: "Use the Bing Maps Elevations API to get elevation information for a set of locations, polyline, or area on the Earth."
```
Comment (stevemunk) [inline BingMaps/rest-services/elevations/index.md:3]:
```suggestion
description: The overview page for the Bing Maps Elevations API, used to get elevation information for a set of locations, polyline or area on the Earth, contains links to and descriptions of the elevations API.
```

File: BingMaps/rest-services/elevations/point-compression-algorithm.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Point Compression Algorithm | Microsoft Docs"
+description: "This article provides step-by-step instructions that describe the point compression algorithm, complete with an example."
```
Comment (stevemunk) [inline BingMaps/rest-services/elevations/point-compression-algorithm.md:3]:
```suggestion
description: This article provides step-by-step instructions that describe the point compression algorithm, complete with an example and instructions how to test your algorithm implementation.
```

File: BingMaps/rest-services/examples/walking-route-example.md [hunk 14]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Walking Route Example | Microsoft Docs"
+description: "The example in this article returns a walking route from the Eiffel Tower to the Louvre Museum in Paris, France. Responses are shown for both XML and JSON formats."
```
Comment (stevemunk) [inline BingMaps/rest-services/examples/walking-route-example.md:3]:
```suggestion
description: "The example in this article returns a walking route from the Eiffel Tower to the Louvre Museum in Paris, France, optimized for distance. Responses are shown for both XML and JSON formats."
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by stevemunk** [comment]
@AqMoh Please let me know when you are finished making updates, thanks!

**Comment by AqMoh** [comment]
@stevemunk 
Changes implemented, save for suggestions that have been marked as outdated. Please let me know if this is good to go, thank you!

**Pull request closed without merging**